### PR TITLE
fix: blog sharing and bug in publish blog

### PR DESCRIPTION
**Pull request by piyushagwl007**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#187 
#186 

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: client/src/hooks/useBlog.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { useState, useEffect, useCallback } from 'react';
 import { getBlog, updateBlog as updateBlogCall } from '../apis/blogs';
 import debounce from '../utils/debounce';
 
-const threeSeconds = 3000;
+const threeSeconds = 500;
```
Comment (nihalanisumit) [inline client/src/hooks/useBlog.js:6]: Sahi kiya

File: client/src/hooks/useVotes.js [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
+  getBlogVotes,
+  getMyVotes,
+  upVoteBlog,
+  downVoteBlog
+} from '../apis/votes';
 import debounce from '../utils/debounce';
 import usePrevious from './usePrevious';
 
-const oneSecond = 1000;
+const oneSecond = 300;
```
Comment (nihalanisumit) [inline client/src/hooks/useVotes.js:11]: LOL

**Review by nihalanisumit — APPROVED** [review]
LGTM

File: client/src/hooks/useBlog.js (hunk 1 above)
Comment (piyushagwl007) [reply client/src/hooks/useBlog.js:6]: Yes it was taking too much time.

File: client/src/hooks/useVotes.js (hunk 2 above)
Comment (piyushagwl007) [reply client/src/hooks/useVotes.js:11]: Haha

**Pull request merged**